quinn-proto/src/packet.rs

```diff
@@ -10,6 +10,8 @@ use crate::{
     ConnectionId,
 };
 
+/// A decoder that decods invariant header from packet
+///
```

Comment:
This change should probably be in the previous commit, where the comment was made a doc comment.

Also:
- Typo: "decodes", not "decods"
- "A decoder that decodes" is redundant. Just say "Decodes a QUIC packet's invariant header" or similar.